### PR TITLE
[lldb][AArch64] Read fpmr register from core files

### PR DESCRIPTION
**Pull request by DavidSpickett**
https://developer.arm.com/documentation/ddi0601/2024-06/AArch64-Registers/FPMR--Floating-point-Mode-Register for details of the register.

### CODE REVIEW

File: lldb/test/API/linux/aarch64/fpmr/TestAArch64LinuxFPMR.py [hunk 1]
```diff
@@ -61,3 +65,19 @@ def test_fpmr_register(self):
...
+        self.runCmd("target create --core corefile")
+
+        self.expect(
+            "register read --all",
+            substrs=[
+                "Floating Point Mode Register",
+                f"fpmr = {self.EXPECTED_FPMR:#018x}",
+            ],
+        )
+        self.expect("register read fpmr", substrs=self.EXPECTED_FPMR_FIELDS)
```
Comment (DavidSpickett) [inline lldb/test/API/linux/aarch64/fpmr/TestAArch64LinuxFPMR.py:83]: Note that for core file debug we aren't sending the register info via XML, so I don't need to skip this if XML support is not enabled.

**Comment by DavidSpickett** [comment]
Last one for FPMR. I'll push the release notes directly.

Thanks for reviewing!

**Comment by DavidSpickett** [comment]
ping!

**Comment by DavidSpickett** [comment]
ping! @omjavaid

**Review by omjavaid — APPROVED** [review]
Looks Good. Sorry about delayed review!

**Comment by DavidSpickett** [comment]
Np. I also realised I have been missing review requests on llvm-zorg for weeks too, including yours.

**Pull request merged**